Sources/DispatchQueue.swift

```diff
@@ -9,17 +9,25 @@ import Foundation
 
 public struct DispatchQueue : DispatchObject, DispatchResumable {
     
-    public let queue: dispatch_queue_t!
-
-    public var rawValue: dispatch_object_t! {
-        return queue
+    public typealias RawValue = dispatch_queue_t
+    
+    @available(*, unavailable, renamed="rawValue")
+    public var queue: RawValue {
+        return rawValue
     }
 
-    public init(raw queue: dispatch_queue_t!) {
-        self.queue = queue
+    public let rawValue: RawValue
+
+    @available(*, unavailable, renamed="DispatchQueue(rawValue:)")
```

Comment:
If possible, group unavailable definitions together, before ```let rawValue```. It quickly becomes hard to read otherwise.